src/app/static/src/tests/components/downloadResults/uploadModal.test.ts

```diff
@@ -75,6 +80,37 @@ describe(`uploadModal `, () => {
         coarseOutput: {} as any
     }
 
+    const completeDownloadResults = {
+        summary: {
+            downloading: false,
+            complete: false,
+            error: null,
+            downloadId: null
+        },
+        spectrum: {
+            downloading: false,
+            complete: true,
+            error: null,
+            downloadId: null
```

Comment:
To clarify the logic:
- `downloading`, `complete` and `error` are mutually exclusive?
- If `complete` is true then `downloadId` should always be non-`null`? (contrary to the example here - which I know is only for testing purposes, but is confusing me)